28Stone_HomeAssignment/Services/CountryService.cs

```diff
@@ -0,0 +1,53 @@
+﻿using _28Stone_HomeAssignment.Models;
+
+namespace _28Stone_HomeAssignment.Services
+{
+    public class CountryService
+    {
+        private static IEnumerable<CountryModel> EuropianUnionCountries(List<CountryModel> country)
```

Comment:
var kā extension sataisīt šitās metodes. + šī klase nodarbojas tikai ar kārtošanu/filtrēšanu..
drīzāk sauktu nevis par servisu bet kkādu filtru  CountryFilter/CountryResultOrderer vai kkā tā.. :? 
padomā vai tā nav jēdzīgāk..